File: WS/C++/Q2.cpp
// Any character is entered by the user; write a program to determine whether the character entered is a capital letter, 
// a small case letter, a digit or a special symbol
#include <iostream>
using namespace std;

int main()
{
    char ch;
    int val;
    cout<<"Enter a Character : ";
    cin>>ch;
    val = int(ch);
    cout<<"The ASCII value of character is "<<val; 
    if (val >= 65 && val<= 90)
    {
        cout<<"\nCharacter is 'Capital' letter";
    }
    else if (val >= 97 && val<= 122)
    {
        cout<<"\nCharacter is 'Small' letter";
    }
    else if (val >= 48 && val<= 57)
    {
        cout<<"\nCharacter is a 'Digit'";
    }
    else if ((val >= 0 && val<= 47) || (val >= 58 && val<= 64) || (val >= 91 && val<= 96) || (val >= 123 && val<= 127) )
    {
        cout<<"\nCharacter is special character";
    }
}

File: WS/Python/Q2.py
# Any character is entered by the user; write a program to determine whether the character entered is a capital letter, 
# a small case letter, a digit or a special symbol
ch = input("Enter Character : ")
val = ord(ch)
print("ASCII value of Character is ",val)

if (val >= 65 and val<= 90):
    print("\nCharacter is 'Capital' letter")
elif (val >= 97 and val<= 122):
    print("\nCharacter is 'Small' letter")
elif (val >= 48 and val<= 57):
    print("\nCharacter is a 'Digit'")
elif ((val >= 0 and val<= 47) or (val >= 58 and val<= 64) or (val >= 91 and val<= 96) or (val >= 123 and val<= 127) ):
    print("\nCharacter is special character")
